testing/mariadb-connector-odbc/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Contributor: Joe Searle <joe@jsearle.net>
+# Maintainer: Joe Searle <joe@jsearle.net>
+pkgname="mariadb-connector-odbc"
+pkgver="3.1.3"
+pkgrel=0
+pkgdesc="MariaDB Connector/ODBC is a standardized, LGPL licensed database driver using the industry standard ODBC API."
+url="https://downloads.mariadb.org/connector-odbc/"
+arch="all"
+license="LGPL-2.1-only"
+depends="unixodbc"
+makedepends="cmake unixodbc unixodbc-dev mariadb-connector-c mariadb-connector-c-dev mariadb-static"
+source="https://downloads.mariadb.org/interstitial/connector-odbc-$pkgver/$pkgname-$pkgver-ga-src.tar.gz 
+	CMakeList.txt.patch"
+builddir="$srcdir/$pkgname-$pkgver-ga-src"
+subpackages="$pkgname-doc"
+
+build() {
+	if [ "$CBUILD" != "$CHOST" ]; then
+		CMAKE_CROSSOPTS="-DCMAKE_SYSTEM_NAME=Linux -DCMAKE_HOST_SYSTEM_NAME=Linux"
+	fi
+	cmake \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DCMAKE_INSTALL_LIBDIR=lib \
+		-DBUILD_SHARED_LIBS=True \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_CXX_FLAGS="$CXXFLAGS" \
+		-DCMAKE_C_FLAGS="$CFLAGS" \
+		${CMAKE_CROSSOPTS} .
+	make
+}
+
```

Comment:
Are there tests? If so, please run them in `check()`, if not please add `options="!check"` to the APKBUILD to indicate that.

Reply:
There are tests in the git repository but they are not included in the tagged releases.

`options="!check" ` has been added.